master/internal/stream/system.go

```diff
@@ -33,7 +33,8 @@ const (
 //
 // There is one PublisherSet for the whole process.  It has one Publisher per streamable type.
 type PublisherSet struct {
-	Trials *stream.Publisher[*TrialMsg]
+	PgAddress string
```

Comment:
nit (non-blocking): is there a reason we aren't using Address to be consistent?

Reply:
no reason, I can change that

Reply:
Actually thinking about it a little more, I'm renaming it to `DBAddress` to make it clear it goes to the DB and not to a destination address.